resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -351,6 +352,15 @@ ecr.dockerfile.label=Dockerfile:
 ecr.repo.label=ECR Repository:
 ecs.clusters=Clusters
 ecs.container_actions_group.label=Containers
+ecs.execute_command_label=Command
+ecs.execute_command_no_command=Command to execute must be entered
+ecs.execute_command_parameters=ecs execute-command --cluster {0} --task {1} --command {2} --interactive
+ecs.execute_command_run=Run Command in Container
+ecs.execute_command_run_command_in_shell=Open Shell
+ecs.execute_command_shell=Shell
```

Comment:
If these are labels in the UI, please append `.label` and they should have a colon after it

Reply:
done